npc/other/Global_Functions.txt

```diff
@@ -434,7 +434,14 @@ function	script	F_ShuffleNumbers	{
 //== Function F_MesColor ===================================
 // Function to colorize npc dialog without having to memorize the color code
 // Examples:
-//    mes callfunc("F_MesColor", C_BLUE) +"This message is now in BLUE";
+//    mes F_MesColor(C_BLUE) +"This message is now in BLUE.";
 function	script	F_MesColor	{
 	return sprintf("^%06X", min(getarg(0), 0xFFFFFF));
 }
+
+//== Function F_MesItemInfo ===================================
+// Show the item name and a clickable link for the item description
+// Only works with mes and mesf, does not work in menu
+function	script	F_MesItemInfo	{
+	return "<ITEM>"+ getitemname(getarg(0)) +"<INFO>"+ getarg(0) +"</INFO></ITEM>";
```

Comment:
Is it good practice to use `" + variable + "` in scripts now?
Also should include a default argument maybe?

`return(sprintf("<ITEM>%s<INFO>%d</INFO></ITEM>", getitemname(getarg(0, "Apple")), getarg(0, Apple)));`

Reply:
suggestion/preference. not suggesting deprecation, rather should be the goto. because stuff like this is hard to read.

`setd(currentquest$+"_"+.@currentmob+"_"+.@x+"_killcount", 0);`

as a newbie i found this harder to comprehend than simply looking at printf stuff, seeing that this substitutes into %s and such.

Reply:
like this
```
setd "currentquest_"+ .@currentmob +"_"+ .@x +"_killcount", 0 ;
-- OR --
setd "currentquest_"+ .@currentmob +"_killcount["+ .@x +"]", 0 ;
setd "currentquest_"+ .@currentmob +"_questname$", 0 ;
```
for me this sentence is like my 2nd language,
since I started scripting since 2006 without programming background

blame the script engine ... 
we don't support 2 dimension array at the moment,
although [I've already suggested it years ago just when I joined hercules](http://herc.ws/board/topic/3886-hercules-1st-2014-megapatch/?do=findComment&comment=25649)

C_ADD on the string was the basic scripting language when athena founded
`*sprintf` only added during rathena period

now I understand when you say my script is hard to read

Reply:
I'd prefer if this PR would use a more readable approach like mesf or sprintf as well please.
Also adding parentheses for function calls would be nice too.
Then you're good to go 👍